lib/load.php

```diff
@@ -184,3 +184,6 @@ function gutenberg_is_experiment_enabled( $name ) {
 
 // Data views.
 require_once __DIR__ . '/experimental/data-views.php';
+
+// Client-side media processing.
+require_once __DIR__ . '/experimental/media/load.php';
```

Comment:
I think it would be more efficient to wrap this in `gutenberg_is_experiment_enabled( 'gutenberg-media-processing' )` rather than unconditionally initializing all the hooks and then checking this inside of every hook callback.